pkg/controller/resourcequota/resource_quota_controller.go

```diff
@@ -56,6 +57,33 @@ type InformerFactory interface {
 	Start(stopCh <-chan struct{})
 }
 
+type informerFactory struct {
```

Comment:
this doesn't seem quota-specific, and the next thing we are likely to do with it is to collapse dynamic informer building in garbage collection onto it. can we place this somewhere both GC and quota can use it?

Reply:
Yeah, sounds good to me, I will move it upp.

Reply:
Moved to `pkg/controller/informer_factory.go`.